tests/test_crud.py

```diff
@@ -145,24 +165,24 @@ def test_get_miru_cluster(self):
         self.assertIsNotNone(created_sample)
         self.assertEqual(miru_id,['BC278'])
 
-    def test_get_cgmlst_cluster(self):
-        cgmlst_cluster_dict = {
-            'key': 'SAM001',
-            'cluster': 'BC300'
-        }
-        sample_dict = {
-            'sample_id': 'SAM001',
-            'accession': 'ACC001',
-            'collection_date': datetime.date(1970, 1, 1)            
-        }
-
-        created_sample = crud.create_sample(self.session, sample_dict)
+    #def test_get_cgmlst_cluster(self):
+    #    cgmlst_cluster_dict = {
+    #        'key': 'SAM001',
+    #        'cluster': 'BC300'
+    #    }
+    #    sample_dict = {
+    #        'sample_id': 'SAM001',
+    #        'accession': 'ACC001',
+    #        'collection_date': datetime.date(1970, 1, 1)            
+    #    }
+
+    #    created_sample = crud.create_sample(self.session, sample_dict)
         
-        result = crud.add_sample_to_cgmlst_cluster(self.session,'SAM001', cgmlst_cluster_dict)
+    #    result = crud.add_sample_to_cgmlst_cluster(self.session,'SAM001', cgmlst_cluster_dict)
 
-        cgmlst_id = crud.get_cgmlst_cluster_by_sample_id(self.session, 'SAM001')
+    #    cgmlst_id = crud.get_cgmlst_cluster_by_sample_id(self.session, 'SAM001')
         
-        self.assertEqual(cgmlst_id,['BC300'])
+    #    self.assertEqual(cgmlst_id,['BC300'])
```

Comment:
Does this test fail?

Reply:
yes this was written for the old model. I rewrote a same test that included library model